docs/usage/configuration.rst

```diff
@@ -101,3 +101,15 @@ By setting to `False`, then optional types will always be treated as the wrapped
 .. literalinclude:: /examples/configuration/test_example_7.py
     :caption: Disable Allow None Optionals
     :language: python
+
+Check Factory Fields
+--------------------
+When `__check_model__` is set to `True`, declaring an argument on the factory that doesn't exist on the model will trigger an exception.
```

Comment:
```suggestion
When `__check_model__` is set to `True`, declaring a field on the factory that doesn't exist on the model will trigger an exception.
```

Just to maintain the consistency since the below sentence uses `fields`.

Reply:
Oh, I like the change you made better than what I suggested. It reads better for some reason :P